lib/lita/handlers/github_pinger.rb

```diff
@@ -116,6 +120,51 @@ def act_on_assign(body, response)
         response
       end
 
+      def act_on_pr_opened(body, response)
+        puts "Detected that someone opened a #{type.tr('_', ' ')}."
+        if type.nil?
+          puts 'Neither pull request or issue detected, exiting...'
+          return
+        end
+
+        if config.enable_round_robin
+          puts "round robin is enabled, selecting the next engineer.."
+          redis_key = 'lita-github-pinger:nextreviewer'
+
+          chosen_reviewer = redis.get(redis_key)
+
+          # get the ball rolling with taylor
+          if chosen_reviewer.nil?
+            redis.set(redis_key, 'taylor')
```

Comment:
so..what happens when you're no longer around? i'd prefer this to just pick the first engineer out of the list, that way we're not hardcoded to a value that may not exist in the future.